edweiss-firebase/functions/src/functions/memento/deleteCards.ts

```diff
@@ -3,15 +3,15 @@ import { onAuthentifiedCall } from 'utils/firebase';
 import { CollectionOf } from 'utils/firestore';
 import { fail, ok } from 'utils/status';
 
-export const deleteCard = onAuthentifiedCall(Memento.Functions.deleteCard, async (userId, args) => {
+export const deleteCards = onAuthentifiedCall(Memento.Functions.deleteCards, async (userId, args) => {
```

Comment:
Be careful! I reviewed a PR (#44) where this function as been renamed into onSanitizedCall() so It might bring some merge conflicts! Speak about it with @holoisme

Reply:
Wonderful, thank you so much for telling me. I will talk to Youssef about this so that changes can be made on his PR (ideally) to avoid conflicts.

Reply:
It is done now. I pushed the newest version that contain the merging with what Youssef has done.